charts/reaper-operator/templates/role.yaml

```diff
@@ -72,4 +69,61 @@ rules:
     verbs:
       - get
       - patch
-      - update
\ No newline at end of file
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: leader-election-role
```

Comment:
What are these leader election roles used for?

Reply:
controller-runtime, the main library on which operators/controllers are built, provides leader election. The leader election stuff is generated with scaffolding.

Reply:
👍  thanks